starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -775,7 +775,6 @@ void reportFullyDrawn() {
   @SuppressWarnings("unused")
   @UsedByNative
   public void setCrashContext(String key, String value) {
-    Log.i(TAG, "setCrashContext Called: " + key + ", " + value);
```

Comment:
Seems fine and I don't think this is strictly needed. We should talk about this offline though. The item flagged as PII was just a UserAgent string. If that's PII, the every log we send is PII. That seems wrong.